packages/task/src/browser/task-configurations.ts

```diff
@@ -302,12 +302,8 @@ export class TaskConfigurations implements Disposable {
     /** Adds given task to a config file and opens the file to provide ability to edit task configuration. */
     async configure(token: number, task: TaskConfiguration): Promise<void> {
         const scope = task._scope;
-        if (scope === TaskScope.Global || scope === TaskScope.Workspace) {
-            return this.taskConfigurationManager.openConfiguration(scope);
-        } else if (typeof scope !== 'string') {
-            console.error('Global task cannot be customized');
-            // TODO detected tasks of scope workspace or user could be customized in those preferences.
-            return;
+        if (scope === TaskScope.Global) {
```

Comment:
It is important to keep the `|| scope === TaskScope.Workspace` option and `this.taskConfigurationManager.openconfiguration(scope)` call from the old code to allow users to open the workspace file, but perhaps the error message should be changed to something like 
```
console.warn(`Cannot open configuration file for specified task. No scope matches {scope}.`);
```

Reply:
if we keep the TaskScope.Workspace in the condition, then provided worksspace tasks would not actually get configured, 
the tasks.json file will be simply opened.

see attached gif.
![workspace-open-config](https://user-images.githubusercontent.com/76971376/109872688-c29dc900-7c3a-11eb-9a9d-469566ff0e59.gif)

Reply:
Hm... seems you're right and I was misunderstanding the flow (for one thing, I thought this was a different file than it was...). In any case, the error message needs to be changed because the `Global` case is handled without getting into this branch.

It still looks like we'll run into problems if `scope === TaskScope.Workspace` though? If the current code sees the value `TaskScope.Workspace (=2)`  , it should end up in the error branch and simply `return` without opening anything, shouldn't it?